lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -320,6 +323,8 @@ void addFPBuiltinDecoration(SPIRVModule *BM, Instruction *Inst,
                      convertFloatToSPIRVWord(F));
     }
   } else if (auto *MD = Inst->getMetadata("fpmath")) {
+    if (!AllowFPMaxError)
+      return;
```

Comment:
Should we report an error here like it's done for `llvm.fpbuiltin`?
Also it looks like this change is not tested in the PR.

Reply:
I discussed this with @sarnex, and he said "fpmath" has been used for a long time to represent FP accuracy, so we cannot start reporting errors with it.  

The change to report an error with "llvm.fpbuiltin" is tested with the CHECK_NO_CAPABILITY_ERROR in IntelFPMaxError.ll